lib/graphiti/sideload.rb

```diff
@@ -411,5 +411,20 @@ def infer_resource_class
     def namespace_for(klass)
       Util::Class.namespace_for(klass)
     end
+
+    def evaluate_flag(flag)
+      return false if flag.blank?
+
+      case flag.class.name
```

Comment:
Pretty minor, but could we avoid the `.name` string allocation? Also `when Symbol, Proc` since they do the same thing.

Reply:
I'm not sure how to do that successfully, to be honest. When I do that and change the `when` statements to classes like `String`. `Proc`, and `Symbol` the code fails.

Reply:
weird